include/ibeo_core/ibeo_core.h

```diff
@@ -275,6 +275,29 @@ class ScannerInfo2205
     const uint16_t& offset);
 };
 
+class ScannerInfo2209
+{
+public:
+  uint8_t device_id;
+  uint8_t scanner_type;
+  uint16_t scan_number;
+  float start_angle;
+  float end_angle;
+  NTPTime scan_start_time;
+  NTPTime scan_end_time;
+  NTPTime scan_start_time_from_device;
+  NTPTime scan_end_time_from_device;
+  float scan_frequency;
+  float beam_tilt;
+  uint32_t scan_flags;
```

Comment:
Can you also parse the `scan_flags` into individual bools or values? something like:

    bool ground_detection_performed;
    bool dirt_detection_performed;
    bool clutter_detection_performed;
    bool scan_from_scan_data_fusion;
    MirrorSide mirror_side;

Here's the spec:

Bit 0: Ground detection was performed
Bit 1: Dirt detection was performed
Bit 2: Clutter detection was performed
Bits 3...8: Reserved
Bit 9: Scan is result from scan data fusion
Bit 10: Mirror side
Bits 11..31: Reserved